__tests__/app.test.js

```diff
@@ -47,30 +46,63 @@ describe("GET", () => {
     });
   });
 
-  describe.only("/api", () => {
+  describe("/api", () => {
     //200: should respond with an object with all endpoints
     //
     it("200: should respond with an object", () => {
       //arrange
       return request(app)
-        .get("/api").expect(200)
-        .then(({body}) => {
-          expect( body.endpoints ).toEqual(endpoints);
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.endpoints).toEqual(endpoints);
         });
     });
 
-    it("200: should respond with an object with key of slug", () => {
-        //arrange
-        return request(app)
-          .get("/api").expect(200)
-          .then(({body}) => {
-            expect( body.endpoints ).toEqual(endpoints);
-          });
-      });
+    //was thinking to add in test for keys on each endpooint object, however each doesnt have the same amount of keys
+
+    //new test
+  });
+
+  //tesk 4
+
+  describe("/api/articles/:article_id", () => {
+    it("200: responds with an article object with correct article_id and correct number of keys", () => {
+      //arrange
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          expect(res.body.article.article_id).toBe(1);
+          expect(Object.values(res.body.article).length).toBe(8);
```

Comment:
We could use objectContaining to check the values of the articles that was received, instead of their sum :)